include/rmm/cuda_stream_view.hpp

```diff
@@ -69,41 +66,7 @@ class cuda_stream_view {
   }
 
  private:
-  cudaStream_t stream_;
+  cudaStream_t stream_{};
```

Comment:
I'd explicitly initialize to make it a little more explicit and obvious what this is doing. 
```suggestion
  cudaStream_t stream_{cudaStreamDefault};
```

Reply:
Done.